src/utils/utils.js

```diff
@@ -63,7 +63,14 @@ Utils.buildCCSSFn = function(js, attr) {
 };
 
 Utils.buildAndExecuteFn =  function (js) {
-  return Utils.buildCCSSFn(js, ['$zoom'])(10);
+  return Utils.buildCCSSFn(
+    R.map(
+      R.compose(
+        replace(/feature\['.*'\] (===|>|<|>=|<=) ('?(.+)(?='|\)| &&))/g, 'true'),
```

Comment:
I added this other regex to avoid user conditionals when a static feature is used with them.

It match strings like this:

```
if((8388607 & (1 << ctx.zoom)) && feature['calificacion'] === 'GRAVE'){_value = true;}

if((8388607 & (1 << ctx.zoom)) && feature['calificacion'] === 34){_value = true;}
```

@alonsogarciapablo